js/packages/components/chat/message/Invitation.tsx

```diff
@@ -0,0 +1,216 @@
+import React, { useMemo, useState } from 'react'
+import { useStyles } from '@berty-tech/styles'
+import { Text as TextNative, TouchableOpacity, View } from 'react-native'
+import { Icon, Text } from '@ui-kitten/components'
+import { useClient, useConversation, useOneToOneContact } from '@berty-tech/store/hooks'
+import { Buffer } from 'buffer'
+import { ProceduralCircleAvatar } from '@berty-tech/components/shared-components'
+import { MessageSystemWrapper } from '@berty-tech/components/chat/message'
```

Comment:
```suggestion
import { ProceduralCircleAvatar } from '../../shared-components/ProceduralCircleAvatar'
import { MessageSystemWrapper } from './MessageSystemWrapper'
```
- please use relative imports for files in the same packages, I'm not sure the behavior around importing self is well defined: `import foo from "./bar"` instead of `import foo from "@org/pkg/bar"`
- please avoid recursive imports if you can `message/Invitation.tsx` -> `message` ->  `message/Invitation.tsx` (due to the `MessageSystemWrapper` import), the js ecosystem is smart around this but it can cause unexpected behaviors that are hard to debug
- please try to import only the minimum required: `import { MessageSystemWrapper } from "message/MessageSystemWrapper"` instead of `import { MessageSystemWrapper } from "message"`, this will help with unit testing